test/functional/simple.js

```diff
@@ -579,6 +625,40 @@ describe('Simple tables', function() {
                     utils.testTidFromDate(new Date('2014-08-11 18:43:58-0700')));
             });
         });
+        it('allows getting TTL for individual rows', function() {
+            return router.request({
+                uri: '/restbase.cassandra.test.local/sys/table/simple-table/',
+                method: 'put',
+                body: {
+                    table: 'simple-table',
+                    attributes: {
+                        key: 'ttl_get_test',
+                        tid: utils.testTidFromDate(new Date('2013-08-08 18:43:58-0700')),
+                        body: new Buffer("<p>test<p>"),
+                        _ttl: 300
+                    }
+                }
+            })
+            .then(function(res) {
+                deepEqual(res.status, 201);
+                return router.request({
+                    uri: '/restbase.cassandra.test.local/sys/table/simple-table/',
+                    method: 'get',
+                    body: {
+                        table: 'simple-table',
+                        attributes: {
+                            key: 'ttl_get_test'
+                        },
+                        _withTTL: true
```

Comment:
But it's exposed, so it's not really *internal*, just not documented. Which I think it should be (see my other comment).